### PR TITLE
Release `v1.0.0b6`

### PR DESCRIPTION
**Pull request by sphuber**

### CODE REVIEW

**Comment by ltalirz** [comment]
Thanks a lot @sphuber !!
Perhaps wait until we know how to proceed with the BandsData issue?
https://github.com/aiidateam/aiida-core/issues/3283

I could fix this if you like...

**Comment by sphuber** [comment]
I am fine with holding on, but given that this is not the only problem, but is a more prolific problem across the data export/show implementation, how long do you want to wait? Do you need the bands export specifically for the tutorial?

**Comment by ltalirz** [comment]
> Do you need the bands export specifically for the tutorial?

I would have used the export, yes, but not necessarily `show`.

If the solution is indeed going to be larger, perhaps you are right and we should not hold off. 
Feel free to just go ahead.

**Comment by sphuber** [comment]
Merged the PRs that we wanted to merge and added the corresponding lines to `README.md`. This should be good to go

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
+## v1.0.0b6
+
+### Deprecations
+- To be removed in `v2.0.0`: the `mode` and `encoding` arguments of the `put_object_from_file` of the `Node` class and `Repository` utility mixin [[#3289]](https://github.com/aiidateam/aiida-core/pull/3289)
+
+### Minor bug fixes
+- Allow to construct `SinglefileData` nodes from binary files [[#3285]](https://github.com/aiidateam/aiida-core/pull/3285)
+- Fix eagerness of `verdi setup --help` being overruled [[#3257]](https://github.com/aiidateam/aiida-core/pull/3257)
```
Comment (ltalirz) [inline CHANGELOG.md]:
```suggestion
- Fix eagerness of `verdi setup/quicksetup --help` being overruled [[#3257]](https://github.com/aiidateam/aiida-core/pull/3257)
```

**Pull request merged**